newIDE/app/src/ExpressionAutocompletion/ExpressionAutocompletion.spec.js

```diff
@@ -292,6 +286,16 @@ describe('ExpressionAutocompletion', () => {
           }),
         ])
       );
+      expect(autocompletions).toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
```

Comment:
I think you only have to add the `expect.objectContaining(...)` to the expect above, in the list parameter of `expect.arrayContaining([...])`

Reply:
I'm not sure because it only check a few attributes of the object.